layers/+distributions/spacemacs-base/keybindings.el

```diff
@@ -235,6 +235,7 @@
   "cd" 'spacemacs/close-compilation-window)
 (with-eval-after-load 'compile
   (define-key compilation-mode-map "r" 'recompile)
+  (define-key evil-motion-state-map "gf" 'find-file-at-point)
   (define-key compilation-mode-map "g" nil))
```

Comment:
Just being curious, what is `(define-key compilation-mode-map "g" nil)` doing, is this really needed?